http/graphql-telemetry/src/test/java/io/quarkus/ts/http/graphql/telemetry/TelemetryIT.java

```diff
@@ -32,8 +30,8 @@ public class TelemetryIT {
 
     @Test
     void verifyTelemetry() {
-        Response classic = sendQuery(createQuery("friend(name:\"Aristotle\"){name}"));
-        Response reactive = sendQuery(createQuery("friend_r(name:\"Aristotle\"){name}"));
+        Response classic = Utils.sendQuery(Utils.createQuery("friend(name:\"Aristotle\"){name}"));
```

Comment:
Could you move on the "Utils" static class name to an import as it's nowadays

Reply:
True!